src/app/Scenes/Artwork/Components/ArtworkCuratorsPickIncreasedInterestCollectorSignal.tsx

```diff
@@ -0,0 +1,58 @@
+import { Flex, Text, TrendingIcon, VerifiedIcon } from "@artsy/palette-mobile"
+import { ArtworkCuratorsPickIncreasedInterestCollectorSignal_artwork$key } from "__generated__/ArtworkCuratorsPickIncreasedInterestCollectorSignal_artwork.graphql"
+import { graphql, useFragment } from "react-relay"
+
+interface Props {
+  artwork: ArtworkCuratorsPickIncreasedInterestCollectorSignal_artwork$key
+}
+
+export const ArtworkCuratorsPickIncreasedInterestCollectorSignal: React.FC<Props> = ({
+  artwork,
+}) => {
+  const { collectorSignals } = useFragment(fragment, artwork)
+
+  if (!collectorSignals) {
```

Comment:
better for readability:
```suggestion
  if (!collectorSignals || (!collectorSignals.increasedInterest && !collectorSignals.curatorsPick)) {
    return null
  }
```
then you won't need the check on line 20.